pkg/koordlet/resmanager/cpu_suppress.go

```diff
@@ -81,9 +81,10 @@ func (r *CPUSuppress) RunInit(stopCh <-chan struct{}) error {
 // writeBECgroupsCPUSet writes the be cgroups cpuset by order
 func (r *CPUSuppress) writeBECgroupsCPUSet(paths []string, cpusetStr string, isReversed bool) {
 	var updaters []resourceexecutor.ResourceUpdater
+	eventHelper := audit.V(3).Reason(resourceexecutor.AdjustBEByNodeCPUUsage).Message("update BE group to cpuset: %v", cpusetStr)
```

Comment:
what is the difference between V(3) and V(1)?
we need a standard for this

Reply:
temporarily use level 3 as the agreed-upon level, and define different audit levels based on needs later